### PR TITLE
Bootstrap refactor

### PR DESCRIPTION
**Pull request by Anbranin**
Little fixes.
Looks miles better than master.
Shouldn't break anything.
Please look over.

### CODE REVIEW

**Comment by Anbranin** [comment]
COVERAGE WENT DOWN WAT

**Comment by Anbranin** [comment]
naw it didn't NM

**Comment by Anbranin** [comment]
screen shot 2016-06-03 at 11 27 09 am
Change this

File: app/assets/stylesheets/dashboard.scss [hunk 1]
```diff
@@ -1,11 +1,45 @@
+@for $i from 1 through 48 {
```
Comment (dfaulken) [inline app/assets/stylesheets/dashboard.scss]: Where'd this come from?

File: app/controllers/application_records_controller.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+require 'prawn'
```
Comment (dfaulken) [inline app/controllers/application_records_controller.rb]: Nicer to have a newline after a `require`

File: app/views/application_records/show.pdf.prawn [hunk 3]
```diff
@@ -0,0 +1 @@
+pdf.text "This is some text"
```
Comment (dfaulken) [inline app/views/application_records/show.pdf.prawn]: This isn't needed.

**Comment by dfaulken** [comment]
Some of this code seems like maybe it wasn't written by the commit authors. It'd be nice to track down and attribute the original sources. I'll open an issue for this.

**Pull request merged**